### DISCUSSION TITLE
Console command setclientcharacter casues character to vanish after save and quit on station in multiplayer

### DISCUSSION THREAD

**Discussion by joveice** [FakeFishGames/Barotrauma#14333] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Using the console command setclientcharacter to switch to a bot and back in MP then saving causing the bot to vanish.
Switch to Bot_A, switch back to Original_Player, Save and Quit. Now when loading the save again Bot_A is gone.
Seems to work fine if you end the round, but if you do not end the round and save it happens.
Tested both ending round by docking/starting mission and by vote.
As long as a round has ended after setclientcharacter has been used it is safe to save and quit, but not before.

### Reproduction steps

1. Start a multiplayer campaign.
2. F3
3. setclientcharacter "your name" "bot name"
4. setclientcharacter "your name" "your name"
5. ESC
6. Save and quit
7. Quit campaign
8. Load campaign
9. Load save

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.5.9.1 (Summer Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by greenking13** [comment]
I can confirm the character deletion happens in the current version. v1.11.4.1 (I cannot confirm the same method, but if you leave to next destination as a bot, your original character gets deleted.)

**Reply by SomeRandomNoobKekeke** [reply]
It's always been like that, i guess it's wontfix and i think this is why [Multiplayer crew manager ](https://steamcommunity.com/sharedfiles/filedetails/?id=3584105566) was made

**Comment by Regalis11** [comment]
I could no longer find a way to reproduce the issue, seems to have been fixed?

https://github.com/user-attachments/assets/b6dda905-418b-4f00-bd23-5ce26bcc23c0

**Reply by SomeRandomNoobKekeke** [reply]
He probably meant the case when you don't switch back

What you showed indeed seems fixed, if it was ever broken
